Intersect.Client/Maps/MapInstance.cs

```diff
@@ -717,42 +718,56 @@ public void DestroyVBOs()
 
         public void DrawItemsAndLights()
         {
-            // Draw map item icons.
-            foreach (var itemCollection in MapItems)
+            // Calculate tile and map item dimensions.
+            var tileWidth = (uint)Options.TileWidth;
+            var tileHeight = (uint)Options.TileHeight;
+            var preferredMapItemWidth = Options.Instance.MapOpts.MapItemWidth;
+            var preferredMapItemHeight = Options.Instance.MapOpts.MapItemHeight;
+            var mapItemWidth = preferredMapItemWidth <= 0 ? tileWidth : preferredMapItemWidth;
+            var mapItemHeight = preferredMapItemHeight <= 0 ? tileHeight : preferredMapItemHeight;
```

Comment:
This should be getting done in `MapOptions.Validate()` in the other file, like I said this gets done when it is loaded.

Also switch to `< 1`